activiti-cloud-services-runtime-bundle/activiti-cloud-services-rest-api/src/main/java/org/activiti/cloud/services/rest/api/TaskVariableAdminController.java

```diff
@@ -0,0 +1,29 @@
+package org.activiti.cloud.services.rest.api;
+
+import org.activiti.api.task.model.payloads.CreateTaskVariablePayload;
+import org.activiti.api.task.model.payloads.UpdateTaskVariablePayload;
+import org.activiti.cloud.services.rest.api.resources.VariableInstanceResource;
+import org.springframework.hateoas.MediaTypes;
+import org.springframework.hateoas.Resources;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+@RequestMapping(value = "/admin/v1/tasks/{taskId}/variables",
+        produces = {MediaTypes.HAL_JSON_VALUE, MediaType.APPLICATION_JSON_VALUE})
+public interface TaskVariableAdminController {
+
+	   @RequestMapping(method = RequestMethod.GET)
+	    Resources<VariableInstanceResource> getVariables(@PathVariable String taskId);
+
+	    @RequestMapping(method = RequestMethod.POST)
+	    ResponseEntity<Void> createVariable(@PathVariable String taskId,
+	                                     	@RequestBody CreateTaskVariablePayload createTaskVariablePayload);
+	    
+	    @RequestMapping(method = RequestMethod.PUT)
+	    ResponseEntity<Void> updateVariable(@PathVariable String taskId,
```

Comment:
@salaboy I'm wondering if we should include the variable name tho the path here:
`PUT /admin/v1/tasks/{taskId}/variables/{variableName}`

Reply:
@erdemedeiros are we doing that for other things? for processes and tasks we have the id right? I will leave it to you.. I think that it make some sense as the name in this case is the unique identifier.

Reply:
Yeah, I'll add it to keep consistency with other `PUT` methods.